CRM/Upgrade/Incremental/php/FiveSeventyFour.php

```diff
@@ -25,10 +25,15 @@ class CRM_Upgrade_Incremental_php_FiveSeventyFour extends CRM_Upgrade_Incrementa
    * Upgrade step; adds tasks including 'runSql'.
    *
    * @param string $rev
-   *   The version number matching this function name
+   *   The version number matUPDATE civicrm_navigation SET url = 'civicrm/import/participant?reset=1'
+WHERE url = 'civicrm/event/import?reset=1';ching this function name
```

Comment:
@eileenmcnaughton Somehow some SQL got mixed into this comment?

Reply:
oops - fixed